src/openqaoa-core/openqaoa/backends/wrapper.py

```diff
@@ -90,40 +90,39 @@ class ZNEWrapper(BaseWrapper):
     Parameters
     ----------
     factory: str
-        The name of the object that determines the zero-noise extrapolation method.
+        The name of the zero-noise extrapolation method. Supported values: "Richardson", "Linear", "Poly", "Exp", "PolyExp", "AdaExp", "FakeNodes".
     scaling: str
-        The name of the function for scaling the noise of a quantum circuit.
-    scale_factors: List[str]
+        The name of the function for scaling the noise of a quantum circuit. Supported values: "fold_gates_at_random", "fold_gates_from_right", "fold_gates_from_left".
+    scale_factors: List[int]
         Sequence of noise scale factors at which expectation values should be measured.
-        For AdaExpFactory, just the first element of the list will be considered.
+        For factory = "AdaExp", just the first element of the list will be considered.
     order: int
         Extrapolation order (degree of the polynomial fit). It cannot exceed len(scale_factors) - 1.
-        Just used for PolyFactory and PolyExpFactory.
+        Only used for factory = "Poly" or "PolyExp".
     steps: int
         The number of optimization steps. At least 3 are necessary.
-        Just used for AdaExpFactory.
+        Only used for factory = "AdaExp".
     """
 
     def __init__(self, backend, factory, scaling, scale_factors, order, steps):
         super().__init__(backend)
 
         # only qiskit backends are supported
-        assert( type(backend) == DEVICE_NAME_TO_OBJECT_MAPPER['qiskit.qasm_simulator'] or 
-                type(backend) == DEVICE_NAME_TO_OBJECT_MAPPER['qiskit.shot_simulator'] or
-                type(backend) == DEVICE_NAME_TO_OBJECT_MAPPER['qiskit.statevector_simulator']
-                or type(backend) == QAOAQiskitQPUBackend), "Only Qiskit backends are supported."
-        # for Mitiq integration, is necessary to transpile the circit. Mitiq doesn't support the RZZ gate.
-        self.parametric_circuit = transpile(self.backend.parametric_circuit, basis_gates=["h","rx","cx"])
-
-        assert(factory in available_factories), "Supported factories are: Poly, Richardson, Exp, FakeNodes, Linear, PolyExp, AdaExp"
-        assert(scaling in available_scaling), "Supported scaling methods are: fold_gates_at_random, fold_gates_from_right, fold_gates_from_left"
-        assert(
-            isinstance(scale_factors, list) and 
-            all(isinstance(x, int) and x >= 1 for x in scale_factors)
-            ), "Scale factor must be a list of ints greater or equal to 1"
-            
-        assert(type(order) == int and order >=1),"Order must a int >=1"
-        assert(type(steps) == int),"Order must a int"
+        if(type(backend) not in [DEVICE_NAME_TO_OBJECT_MAPPER['qiskit.qasm_simulator'],  
+                DEVICE_NAME_TO_OBJECT_MAPPER['qiskit.shot_simulator'],
+                QAOAQiskitQPUBackend]):
+            raise ValueError("Only Qiskit backends are supported, with the expection of the StateVector simulator.")
```

Comment:
Are you saying that `'qiskit.statevector_simulator'` is not supported?

typo for exception